snooty/gizaparser/test_extracts.py

```diff
@@ -3,7 +3,7 @@
 from .extracts import GizaExtractsCategory
 from ..types import Diagnostic, Page, EmbeddedRstParser, ProjectConfig
 from ..parser import make_embedded_rst_parser
-from ..util import ast_to_testing_string
+from ..util_test import ast_to_testing_string
```

Comment:
Is there a reason why we're using `ast_to_testing_string` here and not `check_ast_string`?

Reply:
Mainly there's a lot of tests, and I didn't want to go through and redo all of them when they're not broken, since that would be a significant time sink